px-vis-axis.html

```diff
@@ -849,7 +849,7 @@
 
```

Comment:
The approach I was looking at was to just keep the existing functionality and in case of dashPattern substitute the axis marker with an svg one that matches.  This logic could quickly be change to only use svg but thought this a little less invasive.